kitsune/search/v2/documents.py

```diff
@@ -159,7 +175,18 @@ def get_queryset(cls):
 
 
 class AnswerDocument(QuestionDocument):
-    """"""
+    """
+    ES document for Answers. Every Answer in DB gets an AnswerDocument in ES.
+
+    Child class to QuestionDocument and stored in the same index, so AnswerDocument
+    cannot define a field with the same name as one in QuestionDocument.
+    Most QuestionDocument fields are prefixed by "question_".
+    Distingish AnswerDocuments from QuestionDocuments by filtering on whether the document
+    has a non-null "created" field.
```

Comment:
Similar to the comment above. Let's mostly focus on the arch here and the class itself. Not on how to use it